playwright/performance/Initialize.spec.ts

```diff
@@ -0,0 +1,38 @@
+import fetchOmnichannelConfig from '../utils/fetchOmnichannelConfig';
+import fetchTestPageUrl from '../utils/fetchTestPageUrl';
+import { test, expect } from '@playwright/test';
+import OmnichannelEndpoints from '../utils/OmnichannelEndpoints';
+
+const testPage = fetchTestPageUrl();
+const omnichannelConfig = fetchOmnichannelConfig('UnauthenticatedChat');
+
+test.describe('Performance @Performance: ', () => {
```

Comment:
Please remove `:` from `@Performance:`. `@Performance` would be a tag to be used to run these tests.

Reply:
removed